lib/response.js

```diff
@@ -128,8 +132,8 @@ res.send = function send(body) {
     }
 
     deprecate('res.send(status): Use res.sendStatus(status) instead');
-    this.statusCode = chunk;
-    chunk = statusCodes[chunk];
+    this.status(chunk);
+    chunk = statusCodes[this.statusCode] || String(this.statusCode);
```

Comment:
Slightly different behavior here: `"code"` vs `undefined` for invalid codes. Happy to change it back.